### DISCUSSION TITLE
Docfx v3: Is the versioning design working?

### DISCUSSION THREAD

**Discussion by DavidMarquezF** [dotnet/docfx#7976] [Q&A] [answered]
The [versioning design specs ](https://github.com/dotnet/docfx/blob/v3-release/docs/designs/versioning.md) were developed a long time ago. It seems that the objective was to make monikers easier without the need of specifying complex and hard to understand configurations with the groups properties.

https://github.com/dotnet/docfx/issues/3607 was in theory following these specs but I see that you still have to use the groups conifguration.

Is this me not understanding how it works or it hasn't been implemented into v3 yet?

**Comment by yufeih** [comment, marked as answer]
The `groups` config in v3 was implemented to support existing Microsoft Docs projects, we didn't want to introduce much breaking change as part of their migration.

Versioning is working for Microsoft Docs, but for open source static sites, we still need:
- The template to dynamically adjust display of versioned content and a version picker
- Design and produce an output suitable for static file server

**Reply by DavidMarquezF** [reply]
Okay, so then I understand that this is not something I can use yet for my own projects right?

**Reply by yufeih** [reply]
Correct
